be/src/olap/base_tablet.cpp

```diff
@@ -31,6 +31,8 @@ extern MetricPrototype METRIC_query_scan_count;
 BaseTablet::BaseTablet(TabletMetaSharedPtr tablet_meta, DataDir* data_dir)
         : _state(tablet_meta->tablet_state()),
           _tablet_meta(tablet_meta),
+          // TODO: Think we really copy tablet schema here, which cost double mem
+          // cost in TabletManager
```

Comment:
I think it is unnecessary. schema can be got directly from `tablet_meta`. And `tablet_meta` is also saved in tablet.